packages/components/src/table/index.js

```diff
@@ -250,16 +250,22 @@ class TableCard extends Component {
 				</CardBody>
 
 				<CardFooter justify="center">
-					<Pagination
-						key={ parseInt( query.paged, 10 ) || 1 }
-						page={ parseInt( query.paged, 10 ) || 1 }
-						perPage={ rowsPerPage }
-						total={ totalRows }
-						onPageChange={ this.onPageChange }
-						onPerPageChange={ onQueryChange( 'per_page' ) }
-					/>
+					{ isLoading ? (
```

Comment:
Can you please write test for this change.

Reply:
Addressed here https://github.com/woocommerce/woocommerce-admin/pull/7294/commits/6d13e08f6b3369dbfae76aeca1488b90c162e88e